src/main/java/org/folio/repository/ConsortiaEventsLogRepository.java

```diff
@@ -0,0 +1,86 @@
+package org.folio.repository;
+
+import io.vertx.core.Future;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.SqlResult;
+import io.vertx.sqlclient.Tuple;
+import org.folio.rest.jaxrs.model.OutboxEventLog;
+import org.folio.rest.persist.Conn;
+
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+import static org.folio.rest.persist.PostgresClient.convertToPsqlStandard;
+
+public class ConsortiaEventsLogRepository {
+
+  private static final String OUTBOX_TABLE_NAME = "outbox_event_log";
+  private static final String EVENT_ID_FIELD = "event_id";
+  private static final String ENTITY_TYPE_FIELD = "entity_type";
+  private static final String ACTION_FIELD = "action";
+  private static final String PAYLOAD_FIELD = "payload";
+  private static final String INSERT_SQL = "INSERT INTO %s.%s (event_id, entity_type, action, action_date, payload) VALUES ($1, $2, $3, $4, $5)";
+  private static final String SELECT_EVENT_LOGS = "SELECT * FROM %s.%s ORDER BY action_date LIMIT 1000";
```

Comment:
Should an index for action_date be created probably?

Reply:
Thanks, added